deploy.php

```diff
@@ -0,0 +1,300 @@
+<?php
+
+namespace Deployer;
+
+use Symfony\Component\Yaml\Yaml;
+
+require 'recipe/symfony3.php';
+require 'recipe/cachetool.php';
+require 'recipe/sentry.php';
+// TODO DB stuff from deployer-sumo
+
+// Define some variables
+set('client', 'sumocoders');
+set('project', 'forkdeployer');
+set('repository', 'git@github.com:davysumo/forkcms.git');
+set('release', 'current');
+set('production_url', '$productionUrl');
+set('sentry_organization', '$sentryOrganization');
+set('sentry_project_slug', '$sentryProjectSlug');
+set('sentry_token', '$sentryToken');
+
+// Define staging
+host('dev02.sumocoders.eu')
+    ->user('sites')
+    ->stage('staging')
+    ->set('deploy_path', '~/apps/{{client}}/{{project}}')
+    // ->set('branch', 'staging')
+    ->set('branch', '430-replace-capistrano-with-deployer')
+    ->set('bin/php', 'php7.4')
+    ->set('bin/composer', '{{bin/php}} /home/sites/apps/{{client}}/{{project}}/shared/composer.phar')
+    ->set('cachetool', '/var/run/php_74_fpm_sites.sock')
+    ->set('document_root', '~/php74/{{client}}/{{project}}');
+
+// TODO install composer on staging shared
+
+// Define production
+//host('$host')
+//    ->user('sites')
+//    ->stage('production')
+//    ->set('deploy_path', '~/apps/{{client}}/{{project}}')
+//    ->set('branch', 'master')
+//    ->set('bin/php', '$phpBinary')
+//    ->set('cachetool', '$socketPath')
+//    ->set('document_root', '~/php74/{{client}}/{{project}}');
+
+/*************************
+ * No need to edit below *
+ *************************/
+
+set('use_relative_symlink', false);
+
+// Shared files/dirs between deploys
+add('shared_files', ['.env.local', 'app/config/parameters.yml']);
+add('shared_dirs', ['src/Frontend/Files']);
+
+// Writable dirs by web server
+add('writable_dirs', []);
+
+// Disallow stats
+set('allow_anonymous_stats', false);
+
+// Composer
+set('composer_options', '{{composer_action}} --verbose --prefer-dist --no-progress --no-interaction --no-dev --optimize-autoloader --no-suggest');
+
+// Sentry
+set(
+    'sentry',
+    [
+        'organization' => get('sentry_organization'),
+        'projects' => [
+            get('sentry_project_slug'),
+        ],
+        'token' => get('sentry_token')
+    ]
+);
+
+/*****************
+ * Task sections *
+ *****************/
+// Upload tasks
+/**
+ * Install assets from public dir of bundles
+ * @Override from symfony.php - don't know if we need this for Fork
```

Comment:
Ik zou geen taakname overschrijven met andere uitvoering. Als we dan ooit de default nodig hebben gaat het prutsen zijn.

Reply:
@davysumo ?